### PR TITLE
Add more incorrect types for createEvent

### PR DESCRIPTION
**Pull request by ayg**
Taken from Gecko source code.

### CODE REVIEW

File: dom/nodes/Document-createEvent.html [hunk 1]
```diff
@@ -99,48 +101,57 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
-  "MIDIMessageEvent",
+  "KeyEvents",
```
Comment (foolip) [inline dom/nodes/Document-createEvent.html]: This ought to be already covered, since "KeyEvent" above will be pluralized.

File: dom/nodes/Document-createEvent.html [hunk 2]
```diff
@@ -99,48 +101,57 @@
...
   "MediaEncryptedEvent",
   "MediaKeyEvent",
   "MediaKeyMessageEvent",
   "MediaQueryListEvent",
   "MediaStreamEvent",
   "MediaStreamTrackEvent",
+  "MIDIConnectionEvent",
+  "MIDIMessageEvent",
+  "MouseScrollEvents",
```
Comment (foolip) [inline dom/nodes/Document-createEvent.html]: For this and PopUpEvents, maybe list them as non-plural so that "MouseScrollEventss" and "PopUpEventss" aren't tested?

File: dom/nodes/Document-createEvent.html [hunk 3]
```diff
@@ -99,48 +101,57 @@
...
   "TextEvent",
+  "TimeEvent",
   "TrackEvent",
   "TransitionEvent",
   "WebGLContextEvent",
   "WebKitAnimationEvent",
   "WebKitTransitionEvent",
-  "WheelEvent"
+  "WheelEvent",
+  "XULCommandEvent",
```
Comment (cvrebert) [inline dom/nodes/Document-createEvent.html]: Extraneous comma here

Comment (zcorpan) [inline dom/nodes/Document-createEvent.html]: I think trailing comma is OK :-)

Comment (ayg) [inline dom/nodes/Document-createEvent.html]: I always put a trailing comma in lists, so that if someone adds an entry at the end, a) it won't break if they forget to add a comma, b) the diff won't show the last line as changed because of the added comma.  In fact, when writing this patch it initially broke because I added XULCommandEvent without adding the trailing comma!

File: dom/nodes/Document-createEvent.html [hunk 4]
```diff
@@ -99,48 +101,56 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
```
Comment (foolip) [inline dom/nodes/Document-createEvent.html]: Can you revert the changes to keep the ASCIIbetical order? This is what Emacs' sort-lines does, so it must be correct.

Comment (ayg) [inline dom/nodes/Document-createEvent.html]: :!sort in vim uses the locale by default, but I can set LC_ALL=C to do it ASCIIbetically.

Comment (foolip) [inline dom/nodes/Document-createEvent.html]: Different defaults, great :) I guess whichever editor sorted lines first wins in each case.
